### PR TITLE
release note for OLX transcript change

### PR DESCRIPTION
**Pull request by grantgoodmanedX**
## [DOC-3980](https://openedx.atlassian.net/browse/DOC-3980)

Added a section to the Hawthorn release notes about the changes to the <transcript> element for videos in OLX.

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [ ] Subject matter expert:  
- [x] Subject matter expert: @Qubad786 
- [x] Doc team review (sanity check, copy edit, or dev edit?): @edx/doc
- [x] Product review: @sstack22

### Testing

- [x] Ran ./run_tests.sh without warnings or errors

### Post-review

- [ ] Squash commits

### CODE REVIEW

**Comment by debchatigny** [comment]
LGTM - 👍

**Comment by grantgoodmanedX** [comment]
@sstack22 is this what you were looking for?

**Comment by jolyonb** [comment]
Can you expand on this documentation a bit, please? How can I, as an OLX course author, refer to a transcript file that resides in static assets using this mechanism? Further, is the default storage on Amazon s3 handled by having the storage on s3 linked to the edX video ID?

**Comment by jolyonb** [comment]
Just to add some context, I have OLX video tags that have transcripts in static assets that work on edx.org at the moment, but not on the edX mobile app. I don't know if this is an error in my OLX or in the edX app, and my educational technologists pointed me to this documentation update.

**Comment by Qubad786** [comment]
@jolyonb please see if you find [this](https://openedx.atlassian.net/wiki/spaces/EDUCATOR/pages/815366180/Content-store+Deprecation+Summary) useful.

File: en_us/open_edx_release_notes/source/hawthorn.rst [hunk 1]
```diff
@@ -82,6 +82,36 @@ Studio and Course Author Tools
 * You can establish a password policy to require a minimum strength and 
   complexity for passwords. For more information, see `Password Policy`_
 
+Changes to OLX <transcript> Element for Videos
```
Comment (Qubad786) [inline en_us/open_edx_release_notes/source/hawthorn.rst:85]: @grantgoodmanedX I would suggest to include; how should course teams link transcript content files in `/static` directory to video XML if they are building course OLX (as described [here](https://openedx.atlassian.net/wiki/spaces/EDUCATOR/pages/815366180/Content-store+Deprecation+Summary)).

> Transcript content files are present in course OLX `/static` directory and their filename format is "<edx_video_id>-<language_code>.srt". So, for the above OLX, a transcript with the filename "9c563e7d-c86c-4b97-8154-815b421bc80f-en.srt" should be discoverable in course /static directory.

**Comment by jolyonb** [comment]
Yup, this is good thanks.

**Comment by jolyonb** [comment]
Yup, this is good thanks.

**Comment by sstack22** [comment]
@grantgoodmanedX  - thanks for putting this together! I think it looks good, though @Qubad786's comment seems like a good one to add.

**Pull request merged**